README.md

```diff
@@ -284,3 +285,32 @@ Finally, we run and wait for review.
 ```sh
 git pull
 ```
+=======
```

Comment:
Same with this.

Reply:
hmm, it seems to be adding that after rebasing? I'll change it.

Reply:
Sounds good!